src/afl-fuzz-stats.c

```diff
@@ -2438,4 +2444,20 @@ void show_init_stats(afl_state_t *afl) {
 #undef IB
 
 }
+void update_calibration_time(afl_state_t *afl, u64* time){
+  u64 cur =  get_cur_time_us();
+  afl->calibration_time_us = cur-*time;
+  *time = cur;
```

Comment:
ah you overwrite the time parameter, that is why it works.
still - what is the need? 
I would add a fuzz_time to the fuzzer_stats file that calculates the time from run_time  - sync_time - calibration_time - trim_time.